lib/ace/mode/sqlserver_highlight_rules.js

```diff
@@ -147,7 +141,7 @@ var SqlServerHighlightRules = function() {
             regex: "'",
```

Comment:
looks like the `// ' string` rule a little [bellow](https://github.com/ajaxorg/ace/pull/2460/files#diff-de016821e1a0be41f4be155e106e0760R162) is unnecessary.